lib/guardian/plug/test/backdoor.ex

```diff
@@ -0,0 +1,94 @@
+defmodule Guardian.Plug.Test.Backdoor do
+  @moduledoc """
+  This plug allows you to bypass authentication in acceptance tests
+  by passing the token needed to load the current resource directly to your
+  serializer, via a query string parameter.
+
+  ## Installation
+
+  Add the following to your Phoenix router before any other Guardian plugs.
+
+  ```
+  plug Guardian.Plug.Test.Backdoor
+  plug Guardian.Plug.VerifySession
+  ```
+
+  ***This plug is designed for acceptance testing scenarios and should
+  never be added in a production environment.***
+
+  ## Usage
+
+  Now that `Guardian.Plug.Backdoor` is installed, it's time to log in.
+  If you're using [Hound][hound], you can write the following code.
+
+  ```
+  {:ok, token, _} = Guardian.encode_and_sign(user, :access, new_claims)
+  navigate_to "/?token=\#{token}"
+  ```
+
+  If you aren't using [Hound][hound], a simple `GET /?as=User:5` request will
```

Comment:
It looks like there's some spots referencing `as` (here and in a test below). Is this outdated? I don't see any code to support it.

Reply:
@jsteiner You’re right! `as` is changed to `token`. Examples updated.